### PR TITLE
Fix the buildbox (again)

### PR DESCRIPTION
**Pull request by xacrimon**
This PR fixes the CI for now by removing the toolchain file. This isn't optimal but I do not have the bandwidth to debug our CI pipeline further right now.

### CODE REVIEW

**Comment by zmb3** [comment]
What happen to the linter fix and librdprs.h?

File: build.assets/Dockerfile [hunk 1]
```diff
@@ -83,7 +83,6 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
  ENV RUSTUP_HOME=/usr/local/rustup \
      CARGO_HOME=/usr/local/cargo \
      PATH=/usr/local/cargo/bin:$PATH \
-     RUST_VERSION=1.56.1
```
[the remark this answers is not in the document]
Comment (xacrimon) [reply build.assets/Dockerfile:86]: Ah, 👍

**Comment by xacrimon** [comment]
@zmb3 Messed up git somewhere and went to get some other stuff sorted. They're back now.

File: build.assets/Dockerfile [hunk 2]
```diff
@@ -85,18 +85,18 @@ RUN (groupadd ci --gid=$GID -o && useradd ci --uid=$UID --gid=$GID --create-home
...
-     rustup target add arm-unknown-linux-gnueabihf; \
-     rustup target add aarch64-unknown-linux-gnu; \
-     rustup target list | grep installed; \
-     rustc --version;
+COPY --from=rust:1.56.1 /usr/local/rustup /usr/local/rustup
+COPY --from=rust:1.56.1 /usr/local/cargo /usr/local/cargo
+RUN set -eux \
+    rustup --version; \
+    cargo --version; \
+    rustup component add --toolchain 1.56.1-x86_64-unknown-linux-gnu rustfmt clippy; \
```
Comment (zmb3) [inline build.assets/Dockerfile:93]: Do we need this `--toolchain` flag?

Comment (xacrimon) [reply build.assets/Dockerfile:93]: Unfortunately it looks like so. They didn't re-release cargo with the 1.56.1 patch release and without it, it gets confused and installs clippy and rustfmt for the wrong toolchains which makes them fail during the linting stage. There isn't a prebuilt Rust image from the official sources with those components either.

Comment (xacrimon) [reply build.assets/Dockerfile:93]: context: removing the toolchain file causes clippy to stop working on the patch release

Comment (xacrimon) [reply build.assets/Dockerfile:93]: https://drone.teleport.dev/gravitational/teleport/7561

**Review by zmb3 — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Bot.

**Pull request merged**